Sources/muterCore/CLICommands/RunCommand/Steps/DiscoverSourceFiles.swift

```diff
@@ -36,16 +44,23 @@ private extension DiscoverSourceFiles {
                              excludingPathsIn providedExcludeList: [String] = []) -> [String] {
         let excludeList = providedExcludeList + defaultExcludeList
         let subpaths = FileManager.default.subpaths(atPath: path) ?? []
-        return subpaths
-            .exclude(pathsContainingItems(from: excludeList))
-            .include(swiftFiles)
-            .map { path + "/" + $0 }
-            .sorted()
+        return includeSwiftFiles(
+            subpaths
+                .exclude(pathsContainingItems(from: excludeList))
+                .map(curry(append)(path))
+        )
+    }
+    
+    func evaluateExclusive(files: [String],
```

Comment:
this could also use a better name, but i don't know what to call it at the moment

Reply:
`findFilesToMutate`?